content/about/license.md

```diff
@@ -4,6 +4,32 @@ date: 2018-12-29T11:02:05+06:00
 layout: "overview"
 ---
 
-### Coming soon
+# Copyright and License Information
 
+Geographic Resources Analysis Support System (GRASS) is Copyright, 1999-2018 GRASS Development Team, and licensed under terms of the GNU General Public License (GPL). This includes all software, documentation, and associated materials.
 
+GRASS GIS >= Version 5 is free software, you can redistribute it and/or modify it under the terms of the [GNU General Public License](http://www.gnu.org/licenses/sgpl-3.0.html) as published by the [Free Software Foundation](https://www.fsf.org/), either version 2 of the license, or (at your discretion) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the [GNU General Public License](http://www.gnu.org/licenses/gpl-3.0.html) for more details or see the [PDF GPL document](https://grass.osgeo.org/uploads/grass/gpl.pdf).
```

Comment:
```suggestion
This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the [GNU General Public License](http://www.gnu.org/licenses/gpl-3.0.html) for more details.
```